buf.gen.yaml

```diff
@@ -0,0 +1,14 @@
+version: v1beta1
+plugins:
+  - name: go
+    out: gen/go
+    opt: paths=source_relative
+  - name: go-grpc
+    out: gen/go
+    opt: paths=source_relative
+  - name: grpc-gateway
```

Comment:
we can leave out the grpc-gateway for now.  i don't think we're going to need to use it.